test/transforms/test_base.py

```
@@ -201,6 +424,149 @@ def test_random_choice_validates_configuration():
+        for _ in range(6):
+            result = choice(bundle)
+            trace = result.get_applied_transforms()[-1]
+            selections.append(tuple(trace["params"]["selected_indices"]))
+            outputs.append(as_numpy(result["image"]))
+        return selections, outputs
+
+    first_selections, first_outputs = draw_sequence()
+    second_selections, second_outputs = draw_sequence()
+
+    assert first_selections == second_selections
+    for first, second in zip(first_outputs, second_outputs):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)